### ISSUE TITLE
Connection issues with MySQL 8 over SSH

### ISSUE THREAD

**Issue by ArneAmeye** [Sequel-Ace/Sequel-Ace#1765]
- Sequel Ace Version: 4.0.5 (build 20046)
- Sequel Ace Source: App Store
- macOS Version: Ventura 13.4
- Processor Type: Intel
- MySQL Version: 8
- macOS Localization: EN-us

**Description**
When trying to connect to a MySQL 8 database over SSH it fails and opens a model with error/debug info (which is too long for my screen to be closed, however i can select all text via keyboard shortcuts).
I was actually looking as to why we are not able to connect to any of our MySQL 8 databases with Sequel Pro, I have been able to connect via TablePlus which assures me the connection details are OK.

**Steps To Reproduce**
1. Add a SQL connection to a MySQL 8 database over SSH via key file (id_rsa).
2. Press the "connect" button.
3. Modal with debug info opens after failing the connection.

**Expected Behaviour**
Succesfully connect with a MySQL 8 database over SSH.

**Additional Context**
Debug txt file: [Google drive link](https://drive.google.com/file/d/19zOHrd7VVVlAtliKcDWjtJ4pz-tGsBSd/view?usp=sharing )

**Comment by questionlp** [comment]
If the server the MySQL server instance is running on has `netstat` installed, try running:

```bash
netstat -n | grep 3306
```

There should be an entry that has `:3306` and a status of `LISTENING`. If not, MySQL Server could be set up to only allow connections via socket. If that's the case, then you may need to locate the MySQL Server configuration file and see if `skip-networking` is set. If it is, comment it out and add `bind-address = 127.0.0.1` and restart the service to have it listen to a TCP port.

**Comment by ArneAmeye** [comment]
Just a quick follow-up from my side: 
Our company works with a reputable hosting company where we are a registered reseller but we don't have direct access to the server instance, only SSH within the hosting directory itself. 
For now I have passed your suggestions to their support team. I hope to come back with more helpful information soon.

**Comment by ArneAmeye** [comment]
Our hosting company did not provide us with the details you've been asking me to check for however they have narrowed it down on how to use the credentials for connecting with MySQL8 over SSH.

Before we used the MySQL host as 127.0.0.1 as we thought we were in localhost already when connected over SSH. This and some other assumptions were wrong and could not be used this way with our webhosting provider.

Maybe it might be of help for some others so I will leave a picture here with generic connection details (obviously no real credentials) as to how we managed to make the connection:
![Screenshot 2023-06-22 at 09 31 42](https://github.com/Sequel-Ace/Sequel-Ace/assets/43479752/b5e5b9fc-dd5f-4125-9eaf-48e74fc43edf)
